src/Surfnet/StepupMiddleware/MiddlewareBundle/Migrations/InstitutionConfiguration/MappedInstitutionConfiguration.php

```diff
@@ -71,6 +71,7 @@ final class MappedInstitutionConfiguration
      * @param UseRaLocationsOption $useRaLocationsOption
      * @param ShowRaaContactInformationOption $showRaaContactInformationOption
      * @param VerifyEmailOption $verifyEmailOption
+     * @param SelfVetOptoin $setVetPtion
```

Comment:
Nitpick: `SelfVetOptoin $setVetPtion` here be typos :)

Reply:
I'll leave that up to you. Personally I always use the fully namespaced classes in the annotations, so that I don't have to scroll several hundreds lines up to the use-statements to see what class I'm dealing with.. I guess it's personal, and you are probably using a proper IDE that shows you what class is being resolved